src/api/TokenIO.m

```diff
@@ -273,30 +264,34 @@ - (void)_addKeysAndAlias:(TKUnauthenticatedClient *)client
     MemberOperation *addAlias = [MemberOperation message];
     addAlias.addAlias.aliasHash = [TKHasher hashAlias:alias];
     [operations addObject:addAlias];
+
+    MemberOperation *recoverOp = [MemberOperation message];
+    recoverOp.recoveryRules.recoveryRule.primaryAgent = memberId;
```

Comment:
Just checking, whose member id are you setting as the primary agent for the recovery rules? If we are setting Token as the primary recovery agent on initial member creation, this member id should be the id of the verification service signing member, correct? Since verification service signs the MemberRecoveryOperation package in completeRecovery, the id set in the recovery rules should match the id in that signature.